docs/content/components/collection/Listbox/listbox.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: ListBox
+caption: A component for displaying a list of options and allows user to select one or more of them.
+badge: new
+---
+
+The `<Listbox>` helps you to build accessible listbox of components that can be styled as needed.
```

Comment:
Sorry. Maybe it was misunderstanding. Can we write everywhere in the docs ListBox instead of Listbox? Than it is consistent :-) 

e.g. L.7, L.9, L.18...